packages/test-utils/__data__/noA11yIssues.html

```diff
@@ -13,8 +13,7 @@ <h1>This is a test</h1>
         <div role="contentinfo" id="upside-down"></div>
         <!-- cSpell:disable-line -->
         <script>
-            const shadow = document.getElementById('upside-down').attachShadow({ mode: 'open' });
-            shadow.innerHTML = '<h2 id="shadow">SHADOW DOM</h2><ul><li>Shadow Item 1</li></ul>';
+            // This script is not actually executed! Nothing you put here will run.
```

Comment:
Think its used in integ tests run in browser. 
E.g.  https://github.com/salesforce/sa11y/blob/6c900d72c6fad8e5ca94852cf0b2b983939476a6/packages/browser-lib/__tests__/browser-lib.test.ts#L75-L77

Reply:
Is this test executed in CI, or manually? Below it says "All checks have passed - 10 successful checks."

I'm fine with just adding a comment that it won't run in Jest; I just want to prevent others from wasting time debugging like I did. 😆

Reply:
Updated so I'm just adding a comment.

Reply:
Can you also update the PR desc please.

Reply:
And yes all the tests are executed in the CI

Reply:
Done!